docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,534 @@
+    --from-literal=S3_ACCESS_KEY=$ACCESS_KEY \
+    --from-literal=S3_SECRET_KEY=$SECRET_ACCESS_KEY
+
+kubectl --context $CLUSTER_1 -n $CAMUNDA_NAMESPACE_1 create secret generic elasticsearch-env-secret \
+    --from-literal=S3_ACCESS_KEY=$ACCESS_KEY \
+    --from-literal=S3_SECRET_KEY=$SECRET_ACCESS_KEY
+
+kubectl --context $CLUSTER_1 -n $CAMUNDA_NAMESPACE_1_FAILOVER create secret generic elasticsearch-env-secret \
+    --from-literal=S3_ACCESS_KEY=$ACCESS_KEY \
+    --from-literal=S3_SECRET_KEY=$SECRET_ACCESS_KEY
+```
+
+3. Unset environment variables to reduce the risk of potential exposure.
+
+```bash
+unset ACCESS_KEY
+unset SECRET_ACCESS_KEYv
+```
+
+</TabItem>
+<TabItem value="helm-chart" label="Camunda 8 Helm chart prerequisites">
+
+1. Within the cloned repository, navigate to `test/resources/aws/2-region/kubernetes`. This contains a dual-region example setup.
+
+#### Content Elaboration
+
+Our approach is to work with layered helm values files:
+
+- have a base `camunda-values.yml` that is generally applicable for both Camunda installations
+- two overlays that are for region 0 and region 1 installations
+
+##### camunda-values.yml
+
+This forms the base layer that contains the basic required setup, which is applicable to both regions.
+
+:::note
+Note the `global.multiregion.regions` directive indicates the use for two regions. Disable Identity and Optimize as these are currently not supported within this setup and will break when enabled. Please see the [limitations section](#) <!-- TODO: fill link -->.
+:::
+
+##### region0/camunda-values.yml
+
+This overlay contains the multi region identification for the cluster in region 0.
+
+##### region1/camunda-values.yml
+
+This overlay contains the multi region identification for the cluster in region 1.
+
+#### Preparation
+
+The base `camunda-values.yml` requires some adjustments before installing the Helm chart.
+
+1. Adjust the `placeholder` for `ZEEBE_BROKER_CLUSTER_INITIALCONTACTPOINTS` to reflect your current setup. These are the contact points for the brokers to know how to form the cluster. Find more information on what the variable means in [setting up a cluster](../../../../zeebe-deployment/operations/setting-up-a-cluster.md).
+
+<!-- TODO: currently on a different branch - infex-98-docs-changes -->
+
+**Use the python script in the root of the repository `generate_zeebe_initial_contact.py` to generate the replacement string.**
+
+```bash
+python3 generate_zeebe_initial_contact.py
+
+# It will ask you to provide the following values
+# Enter a namespace for cluster in region 0:
+## in our case camunda-london
+# Enter a namespace for cluster in region 1:
+## in our case camunda-paris
+# Enter Helm release name:
+## the way you'll call the Helm release, for example camunda
+# Enter Zeebe cluster size:
+## for a dual-region setup we recommend 8
+```
+
+In our example, the `placeholder` is replaced with the following:
+
+```yaml
```

Comment:
Let's make a more clear explanation that the users should not use these values, they are just for illustration.